saltwater-repl/src/commands.rs

```diff
@@ -0,0 +1,27 @@
+use crate::repl::Repl;
+use std::collections::HashMap;
+
+pub fn default_commands() -> HashMap<&'static str, fn(&mut Repl, &str)> {
+    let mut map = HashMap::<&'static str, fn(&mut Repl, &str)>::new();
+    map.insert("help", help_command);
+    map.insert("h", help_command);
+    map.insert("quit", quit_command);
+    map.insert("q", quit_command);
+    map
+}
+
+fn help_command(_repl: &mut Repl, _args: &str) {
+    print!(
+        "\
+Available commands:
+    {p}help|h    Shows this message
+    {p}quit|q    Quits the repl
+",
+        p = crate::repl::PREFIX
+    );
```

Comment:
Does rustyline have a way to do this automatically? I want to avoid having to keep this in sync with `default_commands`.

Reply:
No, it doesn't event have support for commands, but I can write some code to generate it.